classes/dbutil.php

```diff
@@ -724,10 +724,14 @@ protected static function table_maintenance($operation, $table, $db = null)
 		{
 			if ( ! in_array($type, array('info', 'warning' , 'error')))
 			{
-				$type = \Fuel::L_INFO;
+                                $type = \Fuel::L_INFO;
```

Comment:
Well your editor has saved them to the file as spaces, if you check the raw copy of the code on github you'll see spaces on this line instead of tabs.